src/back_end/back_end_readme.txt

```diff
@@ -0,0 +1 @@
+Here, the back-end applications.
```

Comment:
I don't think we need back_end for this app, so I suggest to replace it with tests and we will add test description.